README.rst

```diff
@@ -6,13 +6,166 @@ App for tracking sprint commitments, vacations and spillovers.
+    b) [~{JIRA_BOT_USERNAME}]: plan `time` per sprint for this task
+    c) [~{JIRA_BOT_USERNAME}]: plan `time` for reviewing this task
+
+**Note**: The `time` should match the following regexp:
+
+.. code::
+
+    (?:(?P<hours>\d+)\s?h.*?)?\s?(?:(?P<minutes>\d+)\s?m.*?)?
+
+, so anything like 1h 30m, 1h or 30m will work. You can use the following directives for overriding the default values:
+
+Dashboard
+~~~~~~~~~
+For each member of the cell this view displays:
+
+1. Assignee, reviewer, upstream and epic-related hours for the present and upcoming sprint.
+2. Tickets that are missing an estimation.
+3. Scheduled vacations for the upcoming sprint, which are retrieved from the Google Calendar events that match `GOOGLE_CALENDAR_VACATION_REGEX`. Partial vacations (based on the time of the meeting for each person’s day) are retrieved from the `GOOGLE_AVAILABILITY_RANGE` of the `GOOGLE_CONTACT_SPREADSHEET` using the `GOOGLE_AVAILABILITY_REGEX`. The minus (-) suffix there indicates that the member’s timezone is UTC-X. This is taken into account if user’s workday spans over two days in UTC (i.e. start hour is after end hour).
+4. Committed time, which is the sum of all estimation hours.
+5. Goal for the next sprint, which is calculated by summing up user's commitments for each day of the sprint and subtracting `SPRINT_HOURS_RESERVED_FOR_MEETINGS` and `Vacation` hours from it.
+6. Remaining time, which is the result of `Goal` - `Committed`.
+
+User's dashboard
+~~~~~~~~~~~~~~~~
+This view shows all assigned (as `Assignee` or `Reviewer 1`) tickets of the user with:
+
+1. Task's key (you can hover over it to see the ticket's name)
+2. User's role
+3. Current status of the ticket
+4. Remaining time for the current user
+5. Sprint indicator (active or future one)
+6. Epic/Story indicator
+
+Caching
+~~~~~~~
+After you refresh the board for the second time, you’ll immediately see cached data **and a spinner showing that it’s being reloaded**. This makes using the dashboard much smoother.
+
+
+Completing the sprints
+~~~~~~~~~~~~~~~~~~~~~~
+In case when users needs to schedule tickets for sprints that haven’t been created yet, they can press the `Create Next Sprint` to create a new one for the currently viewed cell.
+
+Completing the sprints
+~~~~~~~~~~~~~~~~~~~~~~
+To complete the sprint, you need to have `Staff status` permissions.
+The main idea behind this is that they are not shared by cells - you need to have separate sprint for each one. You can press the `Complete Sprint` button on the cell's dashboard to schedule a Celery task with the following pipeline:
+
+1. Upload spillovers.
+    This uploads all spillovers to the `GOOGLE_SPILLOVER_SPREADSHEET`. The following rows are filled in the spreadsheet:
+
+    a) Ticket
+        The key of the ticket.
+    b) Status
+        The status of the ticket at the moment of ending the sprint.
+    c) Sprints
+        The active sprint (the one that is currently being ended).
```

Comment:
@Agrendalath Does this column list multiple sprints or just the active one? If it just lists the active one it would make sense to change the header from "Sprints" to "Sprint".

Reply:
Just the active one. I renamed it.